artifactory/cli.go

```diff
@@ -1317,7 +1317,14 @@ func nugetLegacyCmd(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	nugetCmd.SetBasicCommand(c.Args().Get(0)).SetArgAndFlags(c.String("nuget-args")).
+
+	args := c.String("nuget-args")
+	if args == "" {
+		nugetCmd.SetArgAndFlags([]string{})
+	} else {
+		nugetCmd.SetArgAndFlags(strings.Split(args, " "))
+	}
```

Comment:
Let's move the above if / else statement into a ```toArgsList``` function, and make this function a bit shorter.